eo/CHANGELOG.md

```diff
@@ -6,7 +6,8 @@ Unreleased (latest)
 
 Changes:
 --------
-- ...
+- Update base image version in Dockerfile
+- Add `mamba` cache cleanup to reduce image size
```

Comment:
If you plan to release a new version now, make sure to put those changes in a new version section instead of `Unreleased`. If you do a common version with @ahandan, it is fine to leave it as it is for now

Same comment for nlp changelog.

Reply:
I can release a new one now or wait, I just need a version update so I can open my PR on birdhouse. Do you still have a lot of changes to do ? @ahandan